app/controllers/api/v2/base_controller.rb

```diff
@@ -58,6 +58,20 @@ def root_node_name
                             end
       end
 
+      # may be extended by plugins to add additional permissions
+      def index_node_permissions
+        {
+          :can_create => can_create?,
+          :can_edit => authorized_for(:controller => controller_name, :action => 'edit'),
+        }
+      end
+
+      def index_node_permissions_snippet
```

Comment:
Your code works well and I approve it but what do you think about this variation of the same method. No regex

```ruby
      def index_node_permissions_snippet
        perms = index_node_permissions.map do |key, value|
          "#{key.to_json}: #{value.to_json}"
        end
        perms.join(",")
      end
```

Reply:
That puts true in quotes, so I want to avoid it if possible